iex_trading/IEX_stocks.ipynb

```diff
@@ -0,0 +1,451 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## IEX Stocks\n",
+    "\n",
+    "In the [previous notebook](./IEX_trading.ipynb) we saw how all the\n",
+    "trades on the IEX stock exchange could be interactively visualized over\n",
+    "the course of a whole day (Monday 21st of October 2019). Using\n",
+    "datashader, all the trades are rasterized interactively to reveal the\n",
+    "density of trades via a colormap.\n",
+    "\n",
+    "When viewing visualized a million trades at once for a whole day, it is\n",
+    "extremely difficult to identify individual trades using a global\n",
+    "view. In order to identify particular trades, it is necessary to zoom\n",
+    "into a time window small enough that individual trades can be\n",
+    "distinguished at which point the trade metadata can be inspected using\n",
+    "the Bokeh hover tool.\n",
+    "\n",
+    "What the global visualization helps reveal is the overall pattern of\n",
+    "trades. In this notebook we will focus on interactively revealing the\n",
+    "trading patterns for individual stocks by partitioning on a set of stock\n",
+    "symbols selected with a widget.\n",
+    "\n",
+    "## Loading the data\n",
+    "\n",
+    "First we will load the data as before, convert the integer timestamps\n",
+    "into the correctly offset datetimes before counting the total number of\n",
+    "events:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "import pandas as pd\n",
+    "df = pd.read_csv('./data/IEX_2019-10-21.csv')\n",
+    "df.timestamp = df.timestamp.astype('datetime64[ns]')\n",
+    "df.timestamp -= datetime.timedelta(hours=4)\n",
```

Comment:
UTC adjustment, maybe a small comment?

Reply:
@philippjfr I can add a comment next to the code though I was hoping this was covered by 'First we will load the data as before, convert the integer timestamps into the correctly offset datetimes before counting the total number of events" ('correctly offset' is referring to the UTC adjustment). Would you still like a simple comment such as `# 4 hour UTC offset` here?

Reply:
Up to you I think, "correctly offset datetimes" does indeed cover it.